Source/WebCore/rendering/RenderLayerScrollableArea.h

```diff
@@ -315,6 +318,8 @@ class RenderLayerScrollableArea final : public ScrollableArea, public CanMakeChe
     
     bool m_isRegisteredForAnimatedScroll { false };
 
+    bool m_hasDarkBackground { false };
```

Comment:
I don't think this member is necessary, since it can be derived from `m_backgroundColor`?